e2e/e2e_test.go

```diff
@@ -430,6 +429,87 @@ var _ = Describe("Central", func() {
 		})
 
 	})
+
+	Describe("should be deployed and can be force-deleted", func() {
+		var err error
+
+		centralName := newCentralName()
+		request := public.CentralRequestPayload{
+			Name:          centralName,
+			MultiAz:       true,
+			CloudProvider: dpCloudProvider,
+			Region:        dpRegion,
+		}
+
+		var createdCentral *public.CentralRequest
+		var namespaceName string
+		It("created a central", func() {
+			createdCentral, err = client.CreateCentral(request)
+			GinkgoWriter.Printf("createdCentral = %+v\n", *createdCentral)
+			Expect(err).To(BeNil())
+			namespaceName, err = services.FormatNamespace(createdCentral.Id)
+			Expect(err).To(BeNil())
+			Expect(constants.CentralRequestStatusAccepted.String()).To(Equal(createdCentral.Status))
+
+			Eventually(func() error {
+				_, err := client.GetCentral(createdCentral.Id)
+				return err
+			}).WithTimeout(waitTimeout).WithPolling(defaultPolling).Should(BeNil())
+		})
+
+		It("should transition central's state to ready", func() {
+			Eventually(func() string {
+				return centralStatus(createdCentral, client)
+			}).WithTimeout(waitTimeout).WithPolling(defaultPolling).Should(Equal(constants.CentralRequestStatusReady.String()))
+		})
+
+		It("should be deletable in the control-plane database", func() {
+			err = adminClient.DbDeleteCentral(createdCentral.Id)
+			Expect(err).To(Succeed())
```

Comment:
I personally prefer:
```suggestion
			Expect(err).ToNot(HaveOccurred())
```
The intention of gomega Expects is to be read somewhat similar to natural language and `Expect(err).To(Succeed())` is not really clear to me.